### PR TITLE
Update README.

### PR DESCRIPTION
**Pull request by nevinheintze**
Expand quick-start guide and add clarification around char/varchar.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,28 +126,85 @@ Spanner database will have a name of the form `pg_dump_{DATE}_{RANDOM}`, where
 See the [Troubleshooting Guide](#troubleshooting-guide) for help on debugging
 issues.
 
+HarbourBridge also [generates several files](#files-generated-by-harbourbridge)
+when it runs: a schema file, a report file (with detailed analysis of the
+conversion), and bad data file (if any data was dropped).
```
Comment (larkee) [inline README.md]: > and bad data file

and a bad data file

File: README.md [hunk 2]
```diff
@@ -113,28 +126,85 @@ Spanner database will have a name of the form `pg_dump_{DATE}_{RANDOM}`, where
...
+
+### Verifying Results
+
+Once the tool has completed, you can verify the new database and its content
+using the Google Cloud Console. Go to the [Cloud Spanner Instances
+page](https://console.cloud.google.com/spanner/instances), select your Spanner
+instance, and then find the database created by HarbourBridge and select
+it. This will list the tables created by HarbourBridge. Select a table, and take
+a look at the schema and data for the table. Next, go to the query page, and try
```
Comment (larkee) [inline README.md]: >and data for the table.

for or from?

File: README.md [hunk 3]
```diff
@@ -227,14 +297,15 @@ The HarbourBridge tool maps PostgreSQL types to Spanner types as follows:
...
 
 All other types map to `STRING(MAX)`. Some of the mappings in this table
 represent loss of precision (marked p), dropped autoincrement functionality
-(marked a), differences in treatment of timezones (marked t) and changes in
-storage size (marked s). We discuss each in turn, as well as other limits and
-notes on schema conversion.
+(marked a), differences in treatment of timezones (marked t), differences in
+treatment of fixed-length character types (marked c), and changes in storage
+size (marked s). We discuss each in turn, as well as other limits and notes on
+schema conversion.
```
Comment (larkee) [inline README.md]: > We discuss each in turn, as well as other limits and notes on
schema conversion.

As is, this sentence is confusing. Did you mean:
> We discuss each in turn below, as well as other limits and notes on
schema conversion.

File: README.md [hunk 4]
```diff
@@ -1,14 +1,27 @@
 # HarbourBridge: Turnkey Postgres-to-Spanner Evaluation
-[![cloudspannerecosystem](https://circleci.com/gh/cloudspannerecosystem/harbourbridge.svg?style=svg)](https://circleci.com/gh/cloudspannerecosystem/harbourbridge)
```
Comment (larkee) [inline README.md]: You have removed this again. I assume from the second commit this is not intended

File: README.md (hunk 1 above)
Comment (nevinheintze) [reply README.md]: Done.

File: README.md [hunk 5]
```diff
@@ -1,14 +1,27 @@
...
-Spanner evaluation. It is not intended for production database migration. In
-particular, statements and features in the pg_dump output that don't map
-directly on Spanner features are simply ignored. It is also not intended for
-databases larger than a couple of GB.
+database populated with data from pg_dump.
+
+HarbourBridge is designed to simplify Spanner evaluation, and in particular to
+bootstrap the process by getting moderate-size PostgreSQL datasets into Spanner
+(up to a few GB). Many features of PostgreSQL, especially those that don't map
+directly to Spanner features, are ignored e.g. (non-primary) indexes, functions
```
Comment (lgruen) [inline README.md]: A few extra commas?
"are ignored, e.g. (non-primary) indexes, functions, and sequences"

File: README.md [hunk 6]
```diff
@@ -113,28 +126,85 @@ Spanner database will have a name of the form `pg_dump_{DATE}_{RANDOM}`, where
...
+of the tables created by HarbourBridge could be significantly impaired.
+
+To improve performance, consider adding [Secondary
+Indexes](https://cloud.google.com/spanner/docs/secondary-indexes) to the tables
+created by HarbourBridge, using the existing PostgreSQL indexes as a guide. Also
+consider using [Interleaved
+Tables](https://cloud.google.com/spanner/docs/schema-and-data-model#creating-interleaved-tables)
+to tune performance.
+
+View HarbourBridge as a base set of functionaity for Spanner evalution that can
```
Comment (lgruen) [inline README.md]: typo: functionality

**Pull request merged**